### PR TITLE
Bug 1762572: Bump openshift-elasticsearch-plugin

### PR DESCRIPTION
**Pull request by jcantrill**
Ref: https://bugzilla.redhat.com/show_bug.cgi?id=1762572

Blocked by https://github.com/fabric8io/openshift-elasticsearch-plugin/pull/189

### CODE REVIEW

File: elasticsearch/Dockerfile.centos7 [hunk 1]
```diff
@@ -52,7 +52,7 @@ ADD probe/ ${ES_HOME}/probe/
 ADD init.sh run.sh prep-install.${RELEASE_STREAM} install.sh ${HOME}/
 COPY utils/** /usr/local/bin/
 
-ARG OSE_ES_URL
+ARG OSE_ES_URL=https://github.com/jcantrill/openshift-elasticsearch-plugin/releases/download/openshift-elasticsearch-plugin-5.6.16.2/openshift-elasticsearch-plugin-5.6.16.2.zip
```
Comment (ewolinetz) [inline elasticsearch/Dockerfile.centos7]: should this still be pointing at your repo?

Comment (jcantrill) [reply elasticsearch/Dockerfile.centos7]: yes.  testing that it passes before actually releasing and merging

**Comment by richm** [comment]
eventrouter flake?
/retest

**Comment by richm** [comment]
looks good - ready to release a new version of openshift-elasticsearch-plugin

**Comment by richm** [comment]
lgtm - let me know when downstream is ready and we can merge this

**Comment by richm** [comment]
I think this PR needs https://github.com/openshift/origin-aggregated-logging/pull/1796 in order to pass tests

**Comment by richm** [comment]
multi-tenancy flake
/retest

**Pull request merged**